packages/grafana-data/src/utils/dataFrameHelper.ts

```diff
@@ -200,6 +200,10 @@ export class DataFrameHelper implements DataFrame {
   getFieldByName(name: string): Field | undefined {
     return this.fieldByName[name];
   }
+
+  toJSON() {
```

Comment:
toDTO?  

toJSON i would expect a json string returned

Reply:
`toJSON` gets called by `JSON.stringify()` -- see:  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#toJSON()_behavior

Reply:
ah, did not know that, then it makes perfect sense

Reply:
FYI, same things in on all Vector instances -- when a Vector is written to JSON, it looks like a normal Array